userCrud/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "domain": "www.example.com",
+  "mongodb": {
+    "username": "jitendra",
+    "password": "jitu2121"
+  }
+}
```

Comment:
new line required

Reply:
While using env params, no need of config

Reply:
ok sir